src/openms/source/ANALYSIS/ID/FalseDiscoveryRate.cpp

```diff
@@ -310,6 +366,24 @@ namespace OpenMS
               {
                 continue;
               }
+              const String peptide_sequence = hit.getSequence().toUnmodifiedString();
+              double peptide_fdr;
+              if (meta_value == "decoy")
+              {
+                peptide_fdr = peptide_to_best_decoy_score[peptide_sequence];
+              }
+              else
+              {
+                peptide_fdr = peptide_to_best_target_score[peptide_sequence];
+              }
+              if (q_value)
+              {
+                hit.setMetaValue("peptide q-value", peptide_fdr);
```

Comment:
I would actually vote for main score and put PSM qvalue in metavalue

Reply:
Why? This is a property of the peptide level (which does not exist in the old id datastructure). So psm, peptide and protein score q-value should be imho main scores.

Reply:
But you can only have one main score.
Until this is done in the new data structure I would prefer having peptide FDR as the main score if both exist for a PeptideHit.

Reply:
Maybe we need to discuss htis in a quick video chat but my idea was to:
- calculate peptide level q-values/fdr by default without the need of setting any flag
- keep the PSM score as main score as otherwise we would break all downstream workflows (or how would you do that?)
- maybe already add a filtering option for peptide FDR in the FDR tool

I am open for suggestions but using the peptide FDR/q-value as main score seemed to break a lot of things (if done by default)

Reply:
I would like flags as for the other levels.
If both psm and peptide are active, peptide is going to be the main score and PSM is meta value.
I don't see any issues with it. As I said, I don't think any of our code requires q-values. The only thing that I think needs to be changed is export to mzTab because you need to change the CV term.